wom/src/test/scala/wom/graph/ExpressionAsCallInputSpec.scala

```diff
@@ -52,7 +52,8 @@ class ExpressionAsCallInputSpec extends FlatSpec with Matchers {
     val callNodeWithInputs = callNodeBuilder.build(
       WomIdentifier("foo"),
       CommandTaskDefinitionSpec.oneInputTask,
-      inputDefinitionFold
+      inputDefinitionFold,
+      Set.empty
```

Comment:
You're right, I left this like this to avoid forgetting to set it in the appropriate places during implementation.

FWIW (!SPOILER ALERT!) I think this is all going away in an upcoming PR so probably not worth worrying too much about in this one.